docs/toc.md

```diff
@@ -1,164 +1,93 @@
-### [Using exception filters](essentials/exceptions/exceptions-user-filtered-handlers.md)
-### [Creating user-defined exceptions](essentials/exceptions/exceptions-create-user-defined.md)
-### [Best Practices for Exceptions](essentials/exceptions/exceptions-best-practices.md)
-## [.NET Assembly File Format](concepts/assembly-format.md)
-## [Garbage Collection](concepts/gc-overview.md)
-## [Generic types](concepts/generics.md)
-## [Delegates and lambdas](concepts/delegates-lambdas.md)
-## [LINQ](concepts/linq.md)
-## [Common Type System & Common Language Specification](concepts/common-type-system.md)
-## [Native interoperability](concepts/native-interop.md)
-## [Package management on .NET](packaging/index.md)
-### [🔧 Introducing NuGet](packaging/overview.md)
-### [🔧 Working with NuGet packages](packaging/working-with-nuget-packages.md)
-### [🔧 Creating NuGet packages](packaging/creating-nuget-packages.md)
-### [🔧 Publishing NuGet packages](packaging/publishing-nuget-packages.md)
-## [Collections and Data Structures](essentials/collections/collections-and-data-structures.md)
-### [Selecting a Collection Class](essentials/collections/selecting-a-collection-class.md)
-### [Commonly Used Collection Types](essentials/collections/commonly-used-collection-types.md)
-### [When to Use Generic Collections](essentials/collections/when-to-use-generic-collections.md)
-### [Comparisons and Sorts Within Collections](essentials/collections/comparisons-and-sorts-within-collections.md)
-### [Sorted Collection Types](essentials/collections/sorted-collection-types.md)
-### [Hashtable and Dictionary Collection Types](essentials/collections/hashtable-and-dictionary-collection-types.md)
-### [Thread-Safe Collections](essentials/collections/thread-safe-collections.md)
-#### [BlockingCollection Overview](essentials/collections/threadsafe/blockingcollection-overview.md)
-#### [When to Use a Thread-Safe Collection](essentials/collections/threadsafe/when-to-use-a-thread-safe-collection.md)
-#### [How to: Add and Remove Items from a ConcurrentDictionary](essentials/collections/threadsafe/how-to-add-and-remove-items.md)
-#### [How to: Add and Take Items Individually from a BlockingCollection](essentials/collections/threadsafe/how-to-add-and-take-items.md)
-#### [How to: Add Bounding and Blocking Functionality to a Collection](essentials/collections/threadsafe/how-to-add-bounding-and-blocking.md)
-#### [How to: Use ForEach to Remove Items in a BlockingCollection](essentials/collections/threadsafe/how-to-use-foreach-to-remove.md)
-#### [How to: Use Arrays of Blocking Collections in a Pipeline](essentials/collections/threadsafe/how-to-use-arrays-of-blockingcollections.md)
-#### [How to: Create an Object Pool by Using a ConcurrentBag](essentials/collections/threadsafe/how-to-create-an-object-pool.md)
-## [🔧 Working with dates](essentials/datetime.md)
-## [Numerics in .NET Core](essentials/numericsincore.md)
-
-
+## [.NET Core Tools](core/tools/index.md)
```

Comment:
Is it okay for the TOC to contain articles that don't exist yet? Shouldn't those be at least "there is nothing here yet" stubs?